roles/hosted_engine_setup/tasks/validate_vlan_name.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Get VLAN devices parent and id as naming convention pattern
+  shell: set -eo pipefail && nmcli -g VLAN.PARENT,VLAN.ID device show {{ vlan_if.nic_if.nic }} | paste -sd '.'
+  with_items:
+    - "{{ vlan_list.results | reject('skipped') | list }}"
+  loop_control:
+    loop_var: vlan_if
+  register: vlan_pattern_list
```

Comment:
The CI fails on `301 Commands should not change things if nothing needs doing`
You can fix it by adding `changed_when: true` as in the previous `shell` task (`Collect interface types`) or recommend adding some logic to check if it changed successfully

Reply:
Thank you very much for your comment, it was certainly helpful :)
I've added the suggested fix